client/src/js/actions/AppActions.js

```diff
@@ -0,0 +1,229 @@
+import AppDispatcher from '../dispatcher/appDispatcher';
+import AppConstants from '../constants/appConstants';
+
+const AppActions = { 
+    saveContact(contact){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.SAVE_CONTACT,
+            contact
+        })
+
+    },
+
+   receiveContact(contacts){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.RECEIVE_CONTACT,
+            contacts
+        })
+
+    },
+
+    receiveNumber(number){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.RECEIVE_ALLUSERS_NUMBER,
+            number
+        })
+
+    },
+
+    removeMessage(messageId){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.REMOVE_MESSAGE,
+            messageId
+        })
+
+    },
+
+    archiveMessages(message){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.ARCHIVE_MESSAGE,
+            message
+        })
+    },
+
+    seenMessage(user){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.SEEN_MESSAGE,
+            user
+        })
+    },
+
+     receiveSeenUsers(users){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.RECEIVE_SEEN_USERS,
+            users
+        })
+    },
+
+    openArchive(){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.OPEN_ARCHIVE
+        })
+    },
+
+     closeArchive(){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.CLOSE_ARCHIVE
+        })
+    },
+
+    inboxMessage(messageId){
+        alert('This feauture is not yet available, coming soon')
```

Comment:
Avoid using alert (and other JS pop up) prompts. Some people think it is bad UX and some have popups disabled which means they will not get to see your message.